src/components/input/index.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from "next/image";
+import React, { useId } from "react";
+
+type InputProps = React.InputHTMLAttributes<HTMLInputElement> & {
+    icon?: any;
+    error?: any;
```

Comment:
Using `any` is not a good idea.

```suggestion
    iconUrl?: string;
    error?: string;
```